css/fonts.css

```diff
@@ -0,0 +1,352 @@
+/* latin-ext */
+@font-face {
+    font-family: 'Cutive Mono';
+    font-style: normal;
+    font-weight: 400;
+    src: url(https://fonts.gstatic.com/s/cutivemono/v14/m8JWjfRfY7WVjVi2E-K9H6RMTm6o39ucNvc.woff2) format('woff2');
```

Comment:
Doesn't this mean browsers will still fetch fonts from Google's servers?

Reply:
you are right. This is my bad! Will fix.